odoo/addons/test_pylint/tests/test_pylint.py

```diff
@@ -27,6 +27,12 @@ class TestPyLint(TransactionCase):
         'W0101',  # unreachable code
 
         'mixed-indentation',
+        'deprecated-module',
+    ]
+
+    BAD_MODULES = [
```

Comment:
Also check for other supported debuggers [1]

[1] https://github.com/odoo/odoo/blob/10.0/odoo/tools/debugger.py#L8